CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project are documented here. The format of this docu
 
 ## [Unreleased]
 
+### Changed
+
+- **BREAKING** Base backends were renamed:
+  - `BasicAuthBackend` -> `BaseBasicAuth`
```

Comment:
`Basic` feels redundant to me ;)

Reply:
`Basic` actually refers to *Basic Authentication*, a standard authentication scheme specified in [IETF RFC 7617](https://tools.ietf.org/html/rfc7617). :-)

Reply:
Oh, I see, makes sense that way. 🤔

I was familiar with the concept, but in past experience, this was usually named `HTTP` (eg. in Symfony Auth Framework uses `HttpBasic`).